hello-world/src/App.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+<div>
+  <nav>
+  <router-link to="/HomeView">Home</router-link> |
+<router-link to="/loginPg">login</router-link> |
+<router-link to="/RegistrationPG">Registration</router-link>
+</nav>
+<router-view/>
+</div>
+</template>
+
```

Comment:
Add a script tag from where you should export the component and add the name of the component.